.prettierignore

```diff
@@ -1,34 +1,13 @@
-# dependencies
-/node_modules
-
-# testing
-/coverage
-
-# production
-/build
-
-# misc
-*.html
+# minified code
 *.min.js
 *.min.css
-*.yml
```

Comment:
I found that when `.html` and `.yml` files were not in the ignore file then if I made any changes in them with VSCode set to `format on save` it would reformat those files.  Of course, that could just be something I need to change in my local setup

Reply:
Gotcha. I'll restore these and the two you called out below. We don't want to make your life harder. :)

Reply:
Well now you don't need to cater to me (though I do appreciate when my live is easier 😁 ) I bring it up in case it isn't me and other run into the same issue.

With that said, I didn't intend for GitHub to highlight both lines 23 and 24 below.  The `**/frontend/build/*` file is likely fine to remove. Sorry about that.

I'll approve either way as having it in or out of the ignore file is not a big deal.

Reply:
:-)

Gotcha, makes sense re `**/frontend/build/*`.